code/modules/food_and_drinks/food/snacks_frozen.dm

```diff
@@ -43,6 +43,143 @@
 	tastes = list("ice cream" = 1, "banana" = 1, "a bad joke" = 1)
 	foodtype = FRUIT | DAIRY | SUGAR
 
+
+/////////////
+//ICE CREAM//
+/////////////
+
+/obj/item/reagent_containers/food/snacks/ice_cream_scoop
+	name = "plain ice cream scoop"
+	desc = "Also known as sweet cream; it still makes for a tasty treat."
+	icon = 'icons/obj/kitchen.dmi'
+	icon_state = "icecream_plain"
+	bonus_reagents = list(/datum/reagent/consumable/ice_cream = 2)
+	list_reagents = list(/datum/reagent/consumable/ice_cream = 10)
+	filling_color = "#EDF7DF"
+	tastes = list("ice cream" = 1)
+	foodtype = DAIRY | SUGAR
+
+/obj/item/reagent_containers/food/snacks/ice_cream_scoop/vanilla
+	name = "vanilla ice cream scoop"
+	desc = "The most commonly known ice cream flavor; it has been and still is widely popular."
```

Comment:
```suggestion
	desc = "The most commonly known ice cream flavor; it has bean and still is widely popular."
```
:)

Reply:
to be clear, this review is a joke and you don't need to follow it if you don't want to

Reply:
I love the funny :]